chart/values.yaml

```
@@ -140,9 +140,46 @@ updateStrategy:
 volumeMounts:
   cache:
     mountPath: /var/cache/nginx
+  nginx-config:
+    mountPath: /etc/nginx/nginx.conf
+    subPath: nginx.conf
+  hedera-explorer-config:
+    mountPath: /app/networks-config.json
+    subPath: networks-config.json
 
 # Volume mounts to add to the container. The key is the volume name and the value is the volume definition. Evaluated as a template.
 volumes:
   cache:
     emptyDir: {}
-
+  nginx-config:
+    configMap:
+      name: '{{ include "hedera-explorer.fullname" . }}-config'
+  hedera-explorer-config:
+    configMap:
+      name: '{{ include "hedera-explorer.fullname" . }}-config'
+
+# Enable this to allows /api/ url to reverse proxy to mirror node url you specify
+# mirrorNodeReverseProxy:
+#     mirrorNodeURL: http://127.0.0.1
```

Perhaps use a more generic proxy solution that doesn't specifically mention mirror node?

```suggestion
proxyPass: {}
  # /api: http://127.0.0.1
```